routing/payment_lifecycle.go

```diff
@@ -386,6 +282,132 @@ func (p *shardHandler) launch(rt *route.Route) (*channeldb.HTLCAttemptInfo,
 	return attempt, nil
 }
 
+// collectResult waits for the result for the given attempt to be available
+// from the Switch, then records the attempt outcome with the control tower. A
+// shardOutcome is returned, indicating how to continue shard creation for this
+// payment.
+func (p *shardHandler) collectResult(attempt *channeldb.HTLCAttemptInfo) (
+	*shardOutcome, error) {
+
+	// If this was a resumed attempt, we must regenerate the
```

Comment:
All the comments and line lengths in this new method should be able to be re-flowed.

Reply:
done.